lib/gs-plugin-loader.c

```diff
@@ -2526,12 +2504,12 @@ mount_removed_cb (GVolumeMonitor *volume_monitor,
 		  GMount *mount,
 		  GsPluginLoader *plugin_loader)
 {
+	GList *list_orig;
 	GsPluginLoaderPrivate *priv = gs_plugin_loader_get_instance_private (plugin_loader);
-	GMount *existing_mount = find_mount_in_list (priv->removable_mounts,
-						     mount);
-	if (existing_mount) {
-		priv->removable_mounts = g_list_remove (priv->removable_mounts,
-							existing_mount);
+
+	list_orig = priv->removable_mounts;
+	priv->removable_mounts = g_list_remove (priv->removable_mounts, mount);
+	if (list_orig != priv->removable_mounts) {
```

Comment:
Nitpick: Remove the curly brackets now that it's a one-liner if body.

Reply:
Fixed locally